frontend/src/utils/externalUrls.ts

```diff
@@ -29,9 +29,31 @@ export type LinkName =
   | "doi2"
   | "doi3"
   | "cawp"
-  | "propublica";
+  | "propublica"
+  | "repJohnLewisTweet"
+  | "deniedVoting"
+  | "aafp"
+  | "rwjf"
+  | "childrenInPrison"
+  | "bjsPrisoners"
+  | "bjsCensusOfJails"
+  | "veraGithub"
+  | "prisonPolicy";
 
 export const urlMap: Record<LinkName, string> = {
+  veraGithub: "https://github.com/vera-institute/incarceration-trends",
+  bjsPrisoners:
+    "https://bjs.ojp.gov/library/publications/prisoners-2020-statistical-tables",
+  bjsCensusOfJails:
+    "https://bjs.ojp.gov/library/publications/census-jails-2005-2019-statistical-tables",
+  prisonPolicy: "https://www.prisonpolicy.org/reports/youth2019.html",
+  childrenInPrison: "https://eji.org/issues/children-in-prison/",
+  rwjf: "https://www.rwjf.org/en/library/research/2019/01/mass-incarceration-threatens-health-equity-in-america.html",
+  aafp: "https://www.aafp.org/about/policies/all/incarceration.html",
+  deniedVoting:
+    "https://www.sentencingproject.org/publications/locked-out-2020-estimates-of-people-denied-voting-rights-due-to-a-felony-conviction/",
+  repJohnLewisTweet:
```

Comment:
Were you seeing like a 404 error or something from Playwright? If so; that's because Twitter blocks scripted stuff, so our Playwright tests throw a warning for those URLs. I just checked the URL manually and it looks like it's still working, so i think this is just a false failure. We should probably move the known failures to a "safe list" array like I had before I moved the tests to playwright;